jena-fuseki2/jena-fuseki-main/src/main/java/org/apache/jena/fuseki/main/FusekiServer.java

```diff
@@ -867,6 +874,25 @@ private static boolean argBoolean(Resource r, Property p, boolean dftValue) {
             }
         }
 
+        private static String argString(Resource r, Property p, String dftValue) {
+            try { GraphUtils.atmostOneProperty(r, p); }
+            catch (NotUniqueException ex) {
+                throw new FusekiConfigException(ex.getMessage());
```

Comment:
Is it worth to include the `ex` in the `FusekiConfigException`, to have more context in the error?

Reply:
In this case, the original exception will in the function called, not with a deeper stack. There isn't more context. The effect is to change the name of the exception to `FusekiConfiguration`.